arbos/retryables/retryable.go

```diff
@@ -157,7 +157,7 @@ func (rs *RetryableState) DeleteRetryable(id common.Hash, evm *vm.EVM, scenario
 	_ = retStorage.ClearByUint64(beneficiaryOffset)
 	_ = retStorage.ClearByUint64(timeoutOffset)
 	_ = retStorage.ClearByUint64(timeoutWindowsLeftOffset)
```

Comment:
I've added a comment with explanation, but if it would be better I can add the error checks there. The explenation:

we ignore returned error as we expect that if one ClearByUint64 fails, than all consecutive calls to ClearByUint64 will fail with the same error (not modifying state), and then ClearBytes will also fail with the same error (also not modifying state) - and this one we check and return

Reply:
please let me know what you think

Reply:
Thanks for the explanation! I think it would be a good practice to handle error immediately rather than relying the fact that this will trigger another error below that we handle. It's easier that way to understand for a reader.

You can address it in follow up PR though.